### PR TITLE
ML.NET v 1.5 Update based on dependabot dotnet/samples PRs

### PR DESCRIPTION
**Pull request by luisquintanilla**
Updated the following tutorials based on dependabot dotnet/samples PRs:

File Type | File Name | Published Url
-- | -- | --
Content | docs/machine-learning/tutorials/github-issue-classification.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/github-issue-classification?branch=pr-en-us-18981
Content | docs/machine-learning/tutorials/image-classification.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/image-classification?branch=pr-en-us-18981
Content | docs/machine-learning/tutorials/image-classification-api-transfer-learning.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/image-classification-api-transfer-learning?branch=pr-en-us-18981
Content | docs/machine-learning/tutorials/iris-clustering.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/iris-clustering?branch=pr-en-us-18981
Content | docs/machine-learning/tutorials/movie-recommendation.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/movie-recommendation?branch=pr-en-us-18981
Content | docs/machine-learning/tutorials/object-detection-onnx.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/object-detection-onnx?branch=pr-en-us-18981
Content | docs/machine-learning/tutorials/predict-prices.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/predict-prices?branch=pr-en-us-18981
Content | docs/machine-learning/tutorials/sales-anomaly-detection.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/sales-anomaly-detection?branch=pr-en-us-18981
Content | docs/machine-learning/tutorials/sentiment-analysis.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/sentiment-analysis?branch=pr-en-us-18981
Content | docs/machine-learning/tutorials/text-classification-tf.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/text-classification-tf?branch=pr-en-us-18981
Content | docs/machine-learning/tutorials/time-series-demand-forecasting.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/time-series-demand-forecasting?branch=pr-en-us-18981

AB#1732831

### CODE REVIEW

File: docs/machine-learning/tutorials/image-classification.md [hunk 1]
```diff
@@ -120,10 +120,10 @@ The training and testing images are located in the assets folders that you'll do
 
     * In Solution Explorer, right-click on your project and select **Manage NuGet Packages**.
     * Choose "nuget.org" as the Package source, select the Browse tab, search for **Microsoft.ML**.
-    * Click on the **Version** drop-down, select the **1.4.0** package in the list, and select the **Install** button.
+    * Select the **Install** button.
```
Comment (briacht) [inline docs/machine-learning/tutorials/image-classification.md:123]: Should we still mention which version (1.5.0)?

Comment (luisquintanilla) [reply docs/machine-learning/tutorials/image-classification.md:125]: Done

**Comment by luisquintanilla** [comment]
@adegeo can I please get a review when you get a chance. Thanks!

**Review by adegeo — APPROVED** [review]
LGTM :shipit:

**Pull request merged**